README.md

```diff
@@ -1,340 +1,535 @@
+For more information about DKG in general, [please visit this page](https://en.wikipedia.org/wiki/Distributed_key_generation).
+
+### DKG tool by SSV
+
+The SSV team built this tool leveraging [drand](https://drand.love/)'s DKG protocol implementation ([please visit their documentation](https://drand.love/docs/cryptography/#setup-phase) for more details on it). This implementation operates under the assumption of a p2p network, allowing operators to communicate.
+The `ssv-dkg` was built to lift this assumption and provide a communication layer that centered on an Initiator figure, to facilitate communication between operators. The introduced potential risk for centralization and bad actors is handled with signatures and signature verifications, as explained in the Security notes section.
+Finally, the outcome of the DKG ceremony is a BLS key pair to be used for validator duties by operators on the ssv.network. As such, the tool ends the process by creating a deposit file to activate the newly created validator key pair, and proceeds to generating the payload for the transaction, and triggering the transaction itself.
+
+## Overview
+
+In order for the DKG protocol to execute successfully:
+
+* all the chosen Operators must be running the `ssv-dkg` tool as Operators
+* separately, an Initiator (one of the Operators, or a separate entity), starts the DKG ceremony by running the ssv-dkg tool with the init parameter
+* the tool automatically exchange data between the interested parties, as outlined in the [Flow Description](#flow-description) section, until the key shares are created
+
+For details on how to run the tool as an Operator, please head over [to this section containing the related instructions](#operator-quick-start).
+Similarly, head over to [this other section](#initiator-quick-start) for instructions on how to launch the tool as the Initiator of the DKG ceremony.
+
+> ℹ️ NOTE: Threshold is computed automatically using 3f+1 tolerance.
+> 
+## Initiator Quick start
+
+### Generate Initiator identity RSA key pair
+
+The Initiator needs to sign all messages exchanged with DKG participants with an RSA key.
+First of all, write down your chosen password in a text file, for example `password`, replacing `<PASSWORD>` with a password of your choosing:
 
-### Build
+```sh
+echo "<PASSWORD>" >> password
+```
 
+#### Generate Initiator key pair from source
```

Comment:
from source? not sure what that means. maybe just generate new encrypted keypair

Reply:
Thanks for spotting this. I think this is mis-labelled. I wanted to provide a "full docker" version of the procedure, and a "full from source" version.
Where "from source" means you clone the repo and build. But this has been later update to use the node, so needs to be changed!